lldb/test/API/commands/target/dump-separate-debug-info/dwo/TestDumpDwo.py

```diff
@@ -130,3 +130,29 @@ def test_dwos_not_loaded_table_output(self):
                 "0x[a-zA-Z0-9]{16}\s+E\s+.*foo\.dwo",
             ],
         )
+
+    @skipIfRemote
+    @skipIfDarwin
+    @skipIfWindows
```

Comment:
This test doesn't actually debug a process right? It looks like you're creating a target, modifying a setting, and then making sure that LLDB can find the dwo files correctly.

If that's the case, this test should totally work on Darwin and Windows because you can debug targets that use DWOs on Darwin and Windows.

Reply:
The standard clang won't emit .dwo files when you build on a mac unless you specify the target triple. Then you also run into cases where someone might not have your architecture enabled in clang so it won't be able to build with the specified triple. I haven't looked at the Makefile for this, but I am guessing it is building with the clang we built, which we know supports the default triple, but we don't know if it will support a specified triple that allows .dwo files to be created. Any darwin clang producing darwin binaries won't make .dwos

Reply:
In that case, we should figure out if we can build the binaries for Linux when on a Darwin or Windows so that we can test this functionality on more than just Linux. I'm nervous about continuing to add functionality that can work on every host but only testing it on some hosts (in this case, dwo support). I work primarily on Darwin hosts and want to make sure I don't break this functionality on accident, ideally before committing a breaking change.

Reply:
I added a symbol-on-demand test to `TestDumpOso` for Darwin. I don't have a Windows machine, so I'm not sure how I would go about testing it there